README.md

```diff
@@ -63,6 +63,7 @@ Operators
 | `$`                       | The root element to query. This starts all path expressions.       |
 | `@`                       | The current node being processed by a filter predicate.            |
 | `*`                       | Wildcard. Available anywhere a name or numeric are required.       |
+| `^`                       | Carrot. Scan parent node.       |
```

Comment:
I guess you meant "Caret" not "Carrot" 🥕. You should fix this typo in the PR title too 😅